ci-operator/config/openshift/cluster-authentication-operator/openshift-cluster-authentication-operator-release-4.20__periodics.yaml

```diff
@@ -0,0 +1,33 @@
+base_images:
+build_root:
+  from_repository: true
+releases:
+  initial:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.20"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.20"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-external-oidc-conformance
+  cron: '@daily'
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      TEST_SUITE: openshift/conformance
```

Comment:
I usually see conformance called explicitly for parallel & serial, should this be split into two separate tests instead?